src/libraries/Native/native-binplace.proj

```diff
@@ -20,6 +20,7 @@
       <BinPlaceItem Include="$(NativeBinDir)*.dbg" />
       <BinPlaceItem Include="$(NativeBinDir)*.dylib" />
       <BinPlaceItem Include="$(NativeBinDir)*.dwarf" />
+      <BinMobileNative Include="@(BinPlaceItem)" />
```

Comment:
I'm pretty sure this will lead to duplicate copies.  If so, we should condition it with $(BinPlaceTestRuntimePack)

Reply:
done